pkg/operator/controller/status_test.go

```diff
@@ -24,8 +24,8 @@ func TestComputeOperatorStatusConditions(t *testing.T) {
 		inputs      testInputs
 		outputs     testOutputs
 	}{
-		{"no operand namespace", testInputs{false, 0, 0}, testOutputs{true, false, true}},
-		{"no ingresscontrollers available", testInputs{true, 0, 0}, testOutputs{false, false, true}},
```

Comment:
It's not clear to me why this test case was wrong.

Also, consider using explicit field labels here. It's extremely difficult to understand these test cases because of the requisite context switching to refer back to the `testInput` and `testOutput` structs.

Reply:
Thank you for reviewing my PR.  I have pushed a new PR to address your comments.  The previous test case was not wrong.  Previously, it was acceptable to have zero active ingress controllers (from a status conditions perspective). This would cause no transition of the available and progressing status conditions.  With Bug 1695210, having no ingress controllers require the progressing condition to be true and the available condition to be false.